Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Use fixed snapshot of Debian to create a deterministic environment.
 # Snapshot tags can be found at https://hub.docker.com/r/debian/snapshot/tags
-ARG debian_snapshot=buster-20200327
-FROM debian/snapshot:${debian_snapshot}
+ARG debian_snapshot=buster-20200720-slim
+FROM debian:${debian_snapshot}
```

Comment:
note that this is not a snapshot image, in particular AFAICT apt-get packages are not guaranteed to be reproducibly installable. Is there no way to make it work with a (perhaps more recent) snapshot image?

Reply:
Hmm - By 'snapshot' in this case, I'm guessing that you meant that the base image has the apt sources are preconfigured to point at the snapshot.debian.org locations? I'll try and tidy this up to be consistent - and find a later snapshot that has the apropriate image in Docker.io so that the cache can be populated.